htdocs/install/mysql/migration/15.0.0-16.0.0.sql

```diff
@@ -304,3 +304,6 @@ UPDATE llx_c_availability SET type_duration = 'w', qty = 4 WHERE code = 'AV_4W';
 ALTER TABLE llx_boxes_def ADD COLUMN fk_user integer DEFAULT 0 NOT NULL;
 
 ALTER TABLE llx_contratdet ADD COLUMN rang integer DEFAULT 0 AFTER info_bits;
+
+ALTER TABLE llx_facture ADD COLUMN fk_input_reason integer NULL DEFAULT NULL AFTER last_main_doc;
```

Comment:
Having this stored on an invoice looks strange to me. 
what is it for ? 
Knowing the way the user contact us ? if yes, it means this must be stored into proposal or order (and it seems it is already stored on this level).

Reply:
@eldy 
If ever you create invoices without any source document, it's interesting to retrieve this information.